### PR TITLE
Dev 2

### PR DESCRIPTION
**Pull request by vinchevskaya**

### CODE REVIEW

File: DEV-2/FindingOfEvenIndices.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+        public FindingOfEvenIndices(string value)
+        {
+            entryString = value;
+        }
+
+        /// <summary>
+        /// This method removes all odd indexes and displays even.
+        /// </summary>
+        public void FindSequenceEvenIndex()
```
Comment (IgorKheidorov) [inline DEV-2/FindingOfEvenIndices.cs]: single responsibility violation

File: DEV-2/FindingOfEvenIndices.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Text;
+
+namespace DEV_2
+{
+    /// <summary>
+    /// This class looks for all even indexes in a string.
+    /// </summary>
+    class FindingOfEvenIndices
```
Comment (IgorKheidorov) [inline DEV-2/FindingOfEvenIndices.cs]: Bad name

**Review by IgorKheidorov — CHANGES_REQUESTED** [review]

File: DEV-2/EntryPoint.cs [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+{
+    /// <summary>
+    /// The entry point to the program.
+    /// </summary>
+    class EntryPoint
+    {
+        static void Main(string[] args)
+        {
+            OnlyEvenElementsOfTheString entryString = new OnlyEvenElementsOfTheString(Console.ReadLine());
+            entryString.FindSequenceEvenIndex();
```
Comment (AlexanderSoroka) [inline DEV-2/EntryPoint.cs]: double gun - double your fun? Why do you call FindSequenceEvenIndex twice?

File: DEV-2/FindingOfEvenIndices.cs [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+        {
+            entryString = value;
+        }
+
+        /// <summary>
+        /// This method generates a string of characters with positive indices.
+        /// </summary>
+        public string FindSequenceEvenIndex()
+        {
+            StringBuilder sb = new StringBuilder();
```
Comment (AlexanderSoroka) [inline DEV-2/FindingOfEvenIndices.cs:23]: bad variable name

**Review by AlexanderSoroka — APPROVED** [review]

**Pull request merged**